contracts/contribute/Contributors.sol

```diff
@@ -336,9 +336,23 @@ contract Contributors is ERC721TokenReceiver {
         _nonce = localNonce + 1;
     }
 
-    function _reDeploy(uint256 serviceId, address stakingInstance, address multisig) internal {
+    function _reDeploy(uint256 serviceId, address stakingInstance, address multisig, bool updateService) internal {
```

Comment:
When re-deploying from one staking contract to another - the `update()` of the service is needed.

Reply:
not sure why